pkg/logging-reconciler/logging_reconciler.go

```diff
@@ -22,22 +23,21 @@ type LoggingReconciler struct {
 	Reconcilers []reconciler.Interface
 }
 
-func (l *LoggingReconciler) Reconcile(ctx context.Context, object client.Object) (result ctrl.Result, err error) {
+func (l *LoggingReconciler) Reconcile(ctx context.Context, lc loggedcluster.Interface) (result ctrl.Result, err error) {
 	// Logging should be enabled when all conditions are met:
-	//   - logging label is set and true on the object
-	//   - object is not being deleted
+	//   - logging label is set and true on the lc
+	//   - lc is not being deleted
 	//   - TODO(theo) global logging flag is enabled
```

Comment:
Maybe move this comment to where the declaration of common.IsLoggingEnabled is.

Reply:
oops, done!